js/entry.js

```diff
@@ -77,6 +77,9 @@ ipc.on(messages.INITIALIZE_WINDOW, (e, mem) => {
   const windowValue = message.windowValue
 
   currentWindow.setWindowId(windowValue.id)
+  if (process.env.NODE_ENV === 'development') {
+    console.debug(`This Window's ID is:`, windowValue.id)
```

Comment:
Just a little something in each window's console.log to aid window debugging (when in dev mode, and set to show debug messages).